docs/core-concepts/route-params.md

```diff
@@ -0,0 +1,156 @@
+# Route Params
+
+When defining your route `path`, you can use a colon `:` to denote dynamic params.
+
+```ts
+import { 
+  Routes,
+} from 'kitbag/router'
+
+const routes = [
+  {
+    name: 'users',
+    path: '/users/:id',
+    component: ...
+  }
+] as const satisfies Routes
+```
+
+This means the route will expect 1+ extra string characters (anything but a forward slash `/`) in order to be considered a match. The value of these extra characters is captured and exposed in the `route.params`
```

Comment:
> anything but a forward slash `/`

Is that true? Or will the regex that's generated match anything? 

On a side note, if my param was a regex that INCLUDED slashes could I match a parameter across slashes?

Reply:
dude! This is why we need docs, I didn't even know how the thing I wrote works lol. You're definitely right, wrote tests to prove it. This is super cool that params are so flexible and regex params aren't restricted. You could basically circumvent our entire route match logic by just having a single param with your own complex regex pattern.